get_source.sh

```diff
@@ -40,6 +40,13 @@ usage() {
 	echo "  -omr-branch       the OpenJ9/omr git branch: openj9"
 	echo "  -omr-sha          a commit SHA for the omr repository"
 	echo "  -omr-reference    a local repo to use as a clone reference"
+	echo "  -openjceplus-repo the OpenJCEPlus repository url"
+	echo "  -openjceplus-branch the OpenJCEPlus git branch"
+	echo "  -openjceplus-sha  a commit SHA for the OpenJCEPlus repository"
+	echo "  -openjceplus-reference a local repo to use as a clone reference"
+	echo "  -with-gskit-bin   the GSKit binary url"
+	echo "  -with-gskit-sdk-bin the GSKIT SDK binary url"
+	echo "  -with-gskit-bin-credential the credential for downloading the GSKit binaries"
```

Comment:
Suggest the `-with` prefix is not needed for these options.
```
	echo "  -gskit-bin        the GSKit binary url"
	echo "  -gskit-sdk-bin    the GSKIT SDK binary url"
	echo "  -gskit-credential the credential for downloading the GSKit and GSKit SDK"
```
This also reflects the fact that the credentials are used for downloading both GSKit and SDK binaries.